diff --git a/rtt/rt-thread/bsp/qemu-riscv-virt64/SConstruct b/rtt/rt-thread/bsp/qemu-riscv-virt64/SConstruct
index bb0d0793..51c3b728 100644
--- a/rtt/rt-thread/bsp/qemu-riscv-virt64/SConstruct
+++ b/rtt/rt-thread/bsp/qemu-riscv-virt64/SConstruct
@@ -32,7 +32,7 @@ stack_size = 4096
 stack_lds = open('link_stacksize.lds', 'w')
 if GetDepend('__STACKSIZE__'): stack_size = GetDepend('__STACKSIZE__')
 stack_lds.write('__STACKSIZE__ = %d;\r\n' % stack_size)
-if GetDepend('RISCV_S_MODE'):  start_addr = int(0x90200000)
+if GetDepend('RISCV_S_MODE'):  start_addr = int(0x80200000)
 else: start_addr = int(0x80000000)
 stack_lds.write('__START_ADDR__ = 0x%x;' % start_addr)
 
diff --git a/rtt/rt-thread/bsp/qemu-riscv-virt64/link_stacksize.lds b/rtt/rt-thread/bsp/qemu-riscv-virt64/link_stacksize.lds
index 07c26169..6b35b627 100644
--- a/rtt/rt-thread/bsp/qemu-riscv-virt64/link_stacksize.lds
+++ b/rtt/rt-thread/bsp/qemu-riscv-virt64/link_stacksize.lds
@@ -1,2 +1,2 @@
 __STACKSIZE__ = 16384;
-__START_ADDR__ = 0x90200000;
\ No newline at end of file
+__START_ADDR__ = 0x80200000;
\ No newline at end of file
